tools/cli/services/create.js

```diff
@@ -1,25 +1,70 @@
-const fs = require('fs');
-const execute = require('../utils/execute');
-const { red } = require('../utils/colors');
-const { join } = require('path');
+const fs = require("fs");
+const execute = require("../utils/execute");
+const { red } = require("../utils/colors");
+const { join } = require("path");
+
+function runNpmInstallInDirectoryAndExitIfFailed(directoryName) {
+  try {
+    execute(
+      `npm install --prefix ${directoryName}`,
+      "npm install dependencies"
+    );
+  } catch {
+    console.log(red(`Failed to install dependencies using \`npm install\`.`));
+    process.exit(1);
+  }
+}
+
+function installPm2WithNpmIfMissingInDirectoryAndExitIfFailed(directoryName) {
+  if (isPm2NotExists()) {
+    installPm2WithNpmInDirectoryAndExitIfFailed(directoryName);
+  }
+}
+
+function isPm2NotExists() {
+  try {
+    execute(`which pm2`, `check if \`pm2\` is installed globally`);
+    return false;
```

Comment:
Should be true.

Reply:
I'm quite sure the logic is correct. `which pm2` throws an exception if the package is not found. Therefore, it should return false if it is successful since the package was found, and the function returns true if it **doesn't** exist.

The name is misleading and counter-intuitive. I changed the function name to `shouldInstallPm2`.